src/forklift/__main__.py

```diff
@@ -92,11 +93,11 @@ def main():
     elif args['lift']:
         if args['<file-path>']:
             if args['--pallet-arg']:
-                cli.start_lift(args['<file-path>'], args['<arg>'])
+                cli.start_lift(args['<file-path>'], args['<arg>'], skip_copy=args['--skip-copy'])
```

Comment:
i think if we pass any more properties we should strongly consider simply passing the args variable only.

Reply:
Agreed. I wondered what the threshold should be.